index.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Southern California R Users Group"
+page-layout: full
+toc: false
+---
+
+```{r}
+#| include: false
+
+events <- readRDS("events/events_past_and_upcoming.rds")
```

Comment:
Use file.path()

I am going to stop comment on this in other files. Can you check them and correct them.